src/ui/layout/search/GrandSearch.vue

```diff
@@ -0,0 +1,121 @@
+<template>
```

Comment:
Our new search bar at the top of the layout. Currently can show two types of search results:

1. Objects (domain objects)
2. Annotations (tags)

Reply:
I heart this.